docs/admin/networking/index.md

```diff
@@ -77,6 +77,64 @@ direct.
 In general, [port forwarded](./port-forwarding.md) web apps are faster than
 dashboard-accessed web apps.
 
+## Latency
+
+Coder measures and reports several types of latency, providing insights into the performance of your deployment. Understanding these metrics can help you diagnose issues and optimize the user experience.
+
+There are three main types of latency metrics for your Coder deployment:
+
+- Dashboard-to-server latency:
+  
+  The Coder UI measures round-trip time to the Coder server using the browser's Performance API.
```

Comment:
Coder server or workspace proxy

Reply:
updated to

```diff
The Coder UI measures round-trip time to the Coder server
+ or workspace proxy
using the browser's Performance API.
```

[L208](https://github.com/coder/coder/pull/16734/files#diff-f2e36a38235181ed6696007214d293eda4ee5e9c5d8ecb9665b799beaeae8e54R208)